File: python/순서 바꾸기.py
def solution(num_list, n):
    answer = []
    list_left  = []
    list_right = []
    for i in range(len(num_list)):
        if(i < n):
            list_left.append(num_list[i])
        else:
            list_right.append(num_list[i])
    answer += (list_right + list_left)
    return answer

File: java/순서 바꾸기.java
import java.util.*;

class Solution {
    public int[] solution(int[] num_list, int n) {
        ArrayList<Integer> answer = new ArrayList<Integer>();
        for (int i = n; i < num_list.length; i++) {
            answer.add(num_list[i]);
        }
        for (int i = 0; i < n; i++) {
            answer.add(num_list[i]);
        }
        int[] result = answer.stream().mapToInt(Integer::intValue).toArray();
        return result;
    }
}